src/components/Settings/index.tsx

```diff
@@ -1,15 +1,60 @@
+import * as React from "react";
+import Popover from "@mui/material/Popover";
 import SettingsIcon from "@mui/icons-material/Settings";
 import IconButton from "@mui/material/IconButton";
```

Comment:
Let's sort these imports.

```suggestion
import * as React from "react";
import IconButton from "@mui/material/IconButton";
import Popover from "@mui/material/Popover";
import SettingsIcon from "@mui/icons-material/Settings";
```